docs/cluster-management/config.md

```diff
@@ -40,6 +40,21 @@ max_instances: 5
 # instance volume size (GB) (default: 50)
 instance_volume_size: 50
 
+# whether the subnets used for EC2 instances should be public or private (default: "public")
+subnet_visibility: public  # must be "public" or "private"
+
+# whether to include a NAT gateway with the cluster (a NAT gateway is necessary when using private subnets)
+# default value is "none" if subnet_visibility is set to "public"; "single" if subnet_visibility is "private"
+nat_gateway: none  # must be "none", "single", and "highly_available" (one NAT gateway per availability zone)
```

Comment:
it might not be obvious what this explanation `(one NAT gateway per availability zone)` refers to, perhaps update this to `(highly_avialable means one NAT gateway per availability zone)`